sdk/core/Azure.Core.Experimental/src/PopTokenRequestContext.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Contains the details of an authentication token request.
+    /// </summary>
+    public readonly struct PopTokenRequestContext
+    {
+        /// <summary>
+        /// Creates a new TokenRequest with the specified scopes.
+        /// </summary>
+        /// <param name="scopes">The scopes required for the token.</param>
+        /// <param name="parentRequestId">The <see cref="Request.ClientRequestId"/> of the request requiring a token for authentication, if applicable.</param>
+        /// <param name="claims">Additional claims to be included in the token.</param>
+        /// <param name="tenantId"> The tenantId to be included in the token request.</param>
+        /// <param name="isCaeEnabled">Indicates whether to enable Continuous Access Evaluation (CAE) for the requested token.</param>
+        /// <param name="proofOfPossessionNonce">The nonce value required for Pop token requests.</param>
+        /// <param name="request"></param>
```

Comment:
Include a description for this param, so that a meaningful description is displayed on the supporting API ref page